knexfile.js

```diff
@@ -0,0 +1,18 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/sprintDb.sqlite3'
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations'
+    }
```

Comment:
You have your migrations in your data directory, but not your seeds. You should be consistent and add your seeds as well:
```javascript
seeds: {
  directory: './data/seeds'
}
```